include/OpenColorIO/OpenColorTransforms.h

```diff
@@ -1070,6 +1070,9 @@ class OCIOEXPORT FileTransform : public Transform
     static const char * GetFormatNameByIndex(int index);
     /// Get the LUT reader extension at index, return empty string if an invalid index is specified.
     static const char * GetFormatExtensionByIndex(int index);
+    /// Returns true if the extension corresponds to a format supported by FileTransform.
+    /// The argument is case-insensitive, and a leading dot, if present, is ignored.
```

Comment:
It might be helpful to add something like the following:

/// Note that FileTransform will attempt all format readers on a given file until it is successful, even files that contain an unsupported extension or no extension. However, this function is useful for applications that want to know which files are likely to be LUT files, based on their extension.

Reply:
I have appended those additional comments to those existing comments in the latest commit.